### DISCUSSION TITLE
Significance of warning message regarding taps...

### DISCUSSION THREAD

**Discussion by johnalanwoods** [Homebrew/discussions#4495] [Getting started] [answered]
### Output of `brew config`

```shell
: 4.0.17
ORIGIN: https://github.com/Homebrew/brew
HEAD: c0f8068573bbf886fdc78d3f9daca54d53e7e49e
Last commit: 27 hours ago
Core tap JSON: 09 May 10:56 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.40.1 => /opt/homebrew/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 12.6.5-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
`Your system is ready to brew.`
```

### Description of issue

I see the following warning in Doctor:

```
Warning: You have an unnecessary local Cask tap.
This can cause problems installing up-to-date casks.
Please remove it by running:
  brew untap homebrew/cask

Warning: You have an unnecessary local Core tap!
This can cause problems installing up-to-date formulae.
Please remove it by running:
 brew untap homebrew/core
```

Is the reason this is unnecessary because of the move to the JSON based API?

I also have some other taps:

```
homebrew/cask-fonts
homebrew/cask-versions
homebrew/services
```

Can I untap them also?

**Comment by SMillerDev** [comment, marked as answer]
> Is the reason this is unnecessary because of the move to the JSON based API?

Yes, exactly this. You don't need the git repository for the main repositories anymore.

**Reply by johnalanwoods** [reply]
perfect thanks, the other taps I have are still required though right?

**Reply by SMillerDev** [reply]
Yes, although the cask ones are moving towards the main cask too

**Reply by johnalanwoods** [reply]
thank you @SMillerDev

**Reply by Linus-XZX** [reply]
Does this mean I can safely untap `homebrew/core` and `homebrew/cask`? Have some formulas and casks installed but configurations should be mostly stock.

**Reply by SMillerDev** [reply]
Yes, since nothing from there will be used by default.

**Reply by SMillerDev** [reply]
Yes, since nothing from there will be used by default.

**Reply by jkleckner** [reply]
After removing the taps, `brew bundle install` promptly re-created them.

**Reply by SMillerDev** [reply]
Did you put `tap "Homebrew/core"` in your brewfile?

**Reply by jkleckner** [reply]
Doh.  Thanks.
